app/views/pair_requests/_form.html.erb

```diff
@@ -11,14 +11,19 @@
     </div>
   <% end %>
 
+  <div>
+    <%= form.label :invitee_id, style: "display: block" %>
+    <%= form.select :invitee_id, User.invitee_select_for(current_user) %>
+  </div>
+
   <div>
     <%= form.label :when, style: "display: block" %>
     <%= form.datetime_field :when %>
   </div>
 
   <div>
     <%= form.label :duration, style: "display: block" %>
-    <%= form.text_field :duration %>
+    <%= form.select :duration, (15..180).step(15).map { |minutes| ["#{minutes} minutes", minutes * 60] } %>
```

Comment:
I think we should just make this either a text_field or like input counter (whatever those number things are)